_posts/2021-04-01-Whack-Game.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Whack Game 
+description: 
+summary: 
+tags:
+---
+Today, I worked on typing club for about 30 minutes, then I sent my progress to Kajari. Then, we worked on a game which was a game where we had a specific amount of time to whack the mole. So, I followed the tutorial on youtube. [This is the youtube tutorial]  (https://www.youtube.com/watch?v=lhNdUVh3qCc) Then, I finished the game and I talked with Kajari through some specific parts of the game code. 
+The first snippet we talked about was:
+```
+in here it moves the mole frequently, and it moves the mole every 500 milliseconds?
```

Comment:
When you are describing what it's doing, use comments, so that you can tell that it's not part of the code. The comments are the two slashes: `// in here it moves the mole frequently, and it moves the mole every 500 milliseconds?`